po/de.po

```diff
@@ -17763,7 +17763,7 @@ msgstr "_Breite"
 
 #: gnucash/gtkbuilder/dialog-preferences.glade:3671
 msgid "Open new tabs _adjacent to current tab"
-msgstr ""
+msgstr "Neue Registerkarten neben der _aktuellen Registerkarte öffnen"
```

Comment:
@CWehli Ich wäre nach der Diskussion https://lists.gnucash.org/pipermail/gnucash-de/2021-June/012157.html eher für 
`Reiter`, in diesem Fall `Reiter neuer Ansichten `…

Ich denke wir sollten auch sowas wie `graphical control element: tab`ins Glossar einfügen.

Reply:
@fellen: kann ich das in Weblate einfach überscheiben?

Reply:
@CWehli Ich denke schon. Probier doch mal!